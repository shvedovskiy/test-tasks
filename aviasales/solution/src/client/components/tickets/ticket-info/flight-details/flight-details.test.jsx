/* eslint-disable no-undef */
import React from 'react';

import FlightDetails from './flight-details';


describe('<FlightDetails />', () => {
  const props = {
    origin: {
      code: 'ORG',
      name: 'Origin Name',
      date: '12 янв 2018, Сб',
      time: '13:00',
    },
    destination: {
      code: 'DST',
      name: 'Destination Name',
      date: '12 янв 2018, Сб',
      time: '19:00',
    },
    stops: '1',
  };
  const flightDetals = shallow(<FlightDetails {...props} />);

  it('renders properly', () => {
    expect(flightDetals).toMatchSnapshot();
  });
});
