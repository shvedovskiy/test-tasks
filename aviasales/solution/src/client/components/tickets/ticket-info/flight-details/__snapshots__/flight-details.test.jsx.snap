// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlightDetails /> renders properly 1`] = `
<styled__FlightDetails>
  <styled__Route>
    <styled__Time>
      13:00
    </styled__Time>
    <styled__PathWithStops>
      <styled__Stops>
        1 пересадка
      </styled__Stops>
      <styled__Path>
        <styled__PathLine />
        <styled__PathPlane />
      </styled__Path>
    </styled__PathWithStops>
    <styled__Time>
      19:00
    </styled__Time>
  </styled__Route>
  <styled__Locations>
    <styled__Location>
      <styled__Name>
        ORG
        , 
        Origin Name
      </styled__Name>
      <styled__Date>
        12 янв 2018, Сб
      </styled__Date>
    </styled__Location>
    <styled__Location>
      <styled__Name>
        Destination Name
         
        DST
      </styled__Name>
      <styled__Date>
        12 янв 2018, Сб
      </styled__Date>
    </styled__Location>
  </styled__Locations>
</styled__FlightDetails>
`;
